MobiusCore/Source/BrokenConnection.swift

```diff
@@ -19,16 +19,24 @@
 
 import Foundation
 
-public enum BrokenConnection<ValueType> {
-    public static func accept(_ value: ValueType) {
+/// A helper to construct `Connection`s that will fail when used.
+///
+/// `BrokenConnection.connection()` is used when functions return `Connection` fail an assertion. The resulting
+/// connection will trigger an assertion whenever its `accept` or `dispose` methods are called.
+public enum BrokenConnection<Value> {
+    static func _accept(_ value: Value) {
```

Comment:
I believe these can actually be private

Reply:
No, because they’re called from BackwardsCompatiblity.swift. Should be in future though. :-)